setup_parallel.sh

```diff
@@ -0,0 +1 @@
+apt install mpich2
```

Comment:
Please move this to `vagrant/setup.sh` along with all of the other package installations.